moon/moon-view.sh

```diff
@@ -1 +1 @@
-python3 -m osgar.tools.lidarview --pose2d app.pose2d --lidar rosmsg.scan --camera rosmsg./scout_1/camera/left/image_raw/compressed --camera2 rosmsg./scout_1/camera/right/image_raw/compressed --lidar-limit 20000 --window-size 1200 660 --title rosmsg.rot $1  
+python3 -m osgar.tools.lidarview --pose2d app.pose2d --lidar rosmsg.scan --camera rosmsg.left_image --camera2 rosmsg.right_image_raw --lidar-limit 20000 --window-size 1200 660 --title rosmsg.rot $1  
```

Comment:
the topic reference probably isn't meant to include the `_raw` portion
```suggestion
python3 -m osgar.tools.lidarview --pose2d app.pose2d --lidar rosmsg.scan --camera rosmsg.left_image --camera2 rosmsg.right_image --lidar-limit 20000 --window-size 1200 660 --title rosmsg.rot $1  
```

Reply:
thanks :)